src/test/regress/sql/subselect.sql

```diff
@@ -614,3 +614,77 @@ drop table tl1;
 drop table tl2;
 drop table tl3;
 drop table tl4;
+
+-- Check support <dxl:TestExpr> node. TestExpr present with IN queries (equivalent =ANY).
+drop table if exists t1, t2, t3;
```

Comment:
`drop table if exists` queries should be between `-- start_ignore` and `-- end_ignore`, because existence of the tables should not affect the test results.